zerver/tornado/event_queue.py

```diff
@@ -968,7 +968,24 @@ def is_sender_client(client: ClientDescriptor) -> bool:
 
     for user_data in users:
         user_profile_id: int = user_data["id"]
-        flags: Collection[str] = user_data.get("flags", [])
+        flags: List[str] = user_data.get("flags", [])
```

Comment:
Does this need to be a `Set` to avoid double-appending certain flags in the compatibility code?

Reply:
We no longer need these compatibility code -- [discussion](https://github.com/zulip/zulip/pull/26382#discussion_r1349311332)